swatch-metrics-hbi/src/test/java/com/redhat/swatch/hbi/events/security/RhIdentityUtils.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.hbi.events.security;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class RhIdentityUtils {
```

Comment:
Kinda the same question as before.  The last I knew, there wasn't a need for calling RBAC since we're not trying to get roles back for anything?

cc @mstead, did something change where we need RBAC for swatch-metrics-hbi now?